apps/crn-server/src/routes/user.route.ts

```diff
@@ -87,18 +87,11 @@ export const userRouteFactory = (
   userRoutes.get<{ userId: string }>(
     '/users/:userId/groups',
     async (req, res) => {
-      const { query, params } = req;
+      const { params } = req;
```

Comment:
we used query before, don't we need this to maintain current functionality?

Reply:
actually it turns out the pagination was broken on this so I decided to remove it from this endpoint entirely, which meant the query string was no longer needed. 

In FE the endpoint is only ever used on the user profile page without pagination anyway. We could revisit once there is a need to paginate.